src/components/PlanterDetail.js

```diff
@@ -78,18 +77,17 @@ const PlanterDetail = (props) => {
 
         if (
           !planterRegistration ||
-          planterRegistration.planterId !== planterId
+          planterRegistration[0].planter_id !== planterId
```

Comment:
This will throw an exception if the previous query returned no `planter_registrations`.
We need to check `planterRegistration.length` first.

Reply:
Oops, thanks for picking this up!